css/BEM-style.md

```diff
@@ -0,0 +1,223 @@
+# BEM (Block Element Modifier) as a naming architecture for CSS styling
+
+### Projected Time
+
+About 30 - 45 minutes
+- 5-10 minutes for Lesson
+- 10 minutes for Guided Practice
+- 10 minutes for Independent Practice
+- 5 minutes for Challenge
+- 10 minutes for Check for Understanding
+
+### Prerequisites
+
+- [HTML lesson](/html/html.md)
+- [CSS Lesssons](/css/css.md)
+- [SaSS Lessons](/sass/sass.md) (Optional)
+  
+### Motivation
+
+On smaller pet projects, the way you organise your CSS or even SaSS styles does't really matter and it's mainly because you never really look back into the project ever again thinking about maybe scaling it or adding new features every time. But in big projects with over a 100 stylesheets and varied components spread across the application, that can raise a serious issue,since changing the styles for any compponent would mean searching for the absurd CSS naming (either id's, classes or combination of both ) you provided maybe a month before. 
+If that's not worse, how about you also finding out that your new styles written lack behind on specificity and your styles were never applied, because some other previously applied class or even id made your previous styles highly specific compared to your new ones.
+
+Well in such cases BEM naming architecture for  styling comes to your rescue for providing an easy approach to name elements with lowest specificity (just by using classes only) and help write a CSS that is not only scalable but also maintanable as well.
+
+**Which companies use BEM naming Architecture?**  
+Web Development and Design agency like [SparkBox](https://seesparkbox.com/) use BEM during their development process. Read more about it [here](https://seesparkbox.com/foundry/bem_by_example).
+
+### Objectives
+
+**Participants will be able to:**
+- Understand what BEM naming architecture is and why it is used.
+- Understand how it helps with writing a scalable and maintanable CSS file.
+- Use BEM naming architecture on a small example project.
+
+### Materials
+
+The best reference/resource to get you started: 
+- [BEM documentaion](http://getbem.com/introduction/) (You can understand the basics of BEM -2 min read). 
+  
+
+### Lessons
+
+- To understand how BEM is used for styling elements, check out this great article on Mr-Frontend Blog: ['How to write better CSS with BEM'](https://mrfrontend.org/2017/10/write-better-css-with-bem/) by Raymon S (10 min read).
```

Comment:
Please include any required materials in the “materials” section above so apprentices can reference it easily.

Reply:
Do you mean moving this article to the Materials section? If so, what should I add to lessons instead of this?

Reply:
No don’t move it, but add the link there as well plz